### PR TITLE
added copy image command for png cell output

### PR DESCRIPTION
**Pull request by amunger**
For https://github.com/microsoft/vscode/issues/129880

image

- The icon only appears when the mimetype is 'image/png' and the jupyter renderer is used.
- icon used: https://icon-sets.iconify.design/codicon/copy/
- [The clipboard API documentation](https://w3c.github.io/clipboard-apis/#writing-to-clipboard) doesn't specifically state that other mime types are not supported, but svg, jpeg and gif all error with `DOMException: MIME type image/jpeg not supported on write.`
    - [ This SO post](https://stackoverflow.com/questions/62909538/is-there-any-way-to-copy-image-to-clipboard-with-pure-javascript-without-librari) suggests using a canvas object to convert jpeg to png to enable copying

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Comment by rchiodo** [comment]
You might have to update package.json to move the version. I don't know that this extension will auto generate a new build number.

**Comment by amunger** [comment]
npm install seems to have re-written the entire lock file. I'll try to look into the changes

File: src/client/render.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import type { RendererContext } from 'vscode-notebook-renderer';
 import { concatMultilineString } from './helpers';
 import { getTransform } from './transforms';
 import { OpenImageInPlotViewer, SaveImageAs, IsJupyterExtensionInstalled } from './constants';
+import { noop } from 'underscore';
```
Comment (DonJayamanne) [inline src/client/render.tsx:12]: I'd suggest creating `const noop = () => {}` in the constants file, thats better than pulling in `underscore`, unless we can just import what we need using  something like `from 'underscore/noop';`

**Review by DonJayamanne — APPROVED** [review]

File: src/client/render.tsx (hunk 1 above)
Comment (amunger) [reply src/client/render.tsx:12]: whoops, didn't check what I imported there. Good call

**Pull request merged**